h2o-core/src/main/java/hex/Model.java

```diff
@@ -1060,7 +1080,7 @@ public ScoringInfo last_scored() {
 
   // Lower is better
   public float loss() {
-    switch (_parms._stopping_metric) {
+    switch (Optional.ofNullable(_parms._stopping_metric).orElse(ScoreKeeper.StoppingMetric.AUTO)) {
```

Comment:
@koniecsveta why do we need to handle the null case now? What behavior changed?

Reply:
in case when stopping_metric = AUTO and stopping_rounds = 0, then it gets evaluated as stopping_metric = null

Reply:
Interesting :) I think the behavior is somewhat strange then. We start with AUTO to turn it into null and then implicitly convert to AUTO?

Maybe it is because the parameter `_stopping_metric` is misused here (which is not fault of your PR)

Reply:
I think your change makes sense it just looks a little suspicious. Not sure how to improve it though.